clippy_lints/src/non_send_fields_in_send_ty.rs

```diff
@@ -118,14 +122,14 @@ impl<'tcx> LateLintPass<'tcx> for NonSendFieldInSendTy {
                         NON_SEND_FIELDS_IN_SEND_TY,
                         item.span,
                         &format!(
-                            "this implementation is unsound, as some fields in `{}` are `!Send`",
+                            "there are some fields in `{}` are not safe to be sent to another thread",
```

Comment:
> There are some fields in XYZ are not

Sounds wrong to me, I would suggest:

```suggestion
                            "some fields in `{}` are not safe to be sent to another thread",
```

or

```suggestion
                            "there are some fields in `{}` which are not safe to be sent to another thread",
```